proto/ord_schema.proto

```diff
@@ -197,6 +169,35 @@ message Compound {
   repeated Feature features = 13;
 }
 
+/**
+ * Compounds may undergo additional preparation before being used in a
+ * reaction after being received from a supplier or vendor. We encourage
+ * the use of the 'preparation' enum when possible, even if the description
+ * is an oversimplification of the full procedure, which can be detailed
+ * in the 'preparation_details' field.
+ */
+message Preparation {
+  enum PreparationType {
+    UNSPECIFIED = 0;
+    // Compound used as received.
+    NONE = 1;
+    // Compound repurified (e.g., recrystallized).
+    REPURIFIED = 2;
+    // Compound sparged, most likely to be the case with solvents.
+    SPARGED = 3;
+    // Moisture removed, e.g., using molecular sieves.
+    DRIED = 4;
+    // Compound synthesized in-house
+    SYNTHESIZED = 5;
+  }
+  oneof kind {
+    PreparationType preparation = 1;
```

Comment:
Will this lead to python code where we're accessing a ```preparation.preparation``` field? I might prefer ```type``` for the field name

Reply:
oops, yes this should be `type`